samples/sensor/hx711/prj.conf

```diff
@@ -0,0 +1,4 @@
+CONFIG_SENSOR=y
+CONFIG_HX711=y
+CONFIG_GPIO=y
+CONFIG_HX711_CALIBRATION_WEIGHT=500
```

Comment:
It likely makes sense to enable logging here, since the sample has no real use if you don't enable it.

```suggestion
CONFIG_HX711_CALIBRATION_WEIGHT=500
CONFIG_LOG=y
```

Reply:
Done.